screen2.0/src/app/downloads/datamatrices.tsx

```diff
@@ -319,202 +372,241 @@ export function DataMatrices() {
-              yAxisProps={{ ticks: (bounds ? oneRange : fiveRange)(yMin, yMax), title: "UMAP-2", fontSize: 40 }}
-              scatterData={[scatterData]}
-              plotAreaProps={{
-                onFreeformSelectionEnd: (_, c) => setBiosamples(c[0].map((x) => fData[x])),
-                onSelectionEnd: (x) => handleSetBounds(x),
-                freeformSelection: selectMode === "select",
-              }}
-            >
-              <Scatter
-                data={scatterData}
-                pointStyle={{ r: bounds ? 8 : 6 }}
-                onPointMouseOver={(i,_)=> setTimeout(() => {
-                  setTooltip(i)
-                }, 100)}
-                onPointMouseOut={() => setTimeout(() => {
-                  setTooltip(-1)
-                }, 100)}
-                onPointClick={(i) => setBiosamples([fData[i]])}
-              />
-              {tooltip !== -1 && (
-                //X and Y attributes added due to error. Not sure if setting to zero has unintended consequences
-                (<Annotation notScaled notTranslated x={0} y={0}>
-                  <rect x={35} y={100} width={740} height={120} strokeWidth={2} stroke="#000000" fill="#ffffffdd" />
-                  <rect x={55} y={120} width={740 * 0.04} height={740 * 0.04} strokeWidth={1} stroke="#000000" fill={(colorBy === "sampleType" ? sampleTypeColors : ontologyColors)[colorBy === "sampleType" ? fData[tooltip].sampleType : fData[tooltip].ontology]} />
-                  <text x={100} y={140} fontSize="26px" fontWeight="bold">
-                    {fData[tooltip].displayname.replace(/_/g, " ").slice(0, 45)}
-                    {fData[tooltip].displayname.length > 45 ? "..." : ""}
-                  </text>
-                  <text x={55} y={185} fontSize="24px">
-                    {fData[tooltip].experimentAccession}
-                  </text>
-                </Annotation>)
+          </Stack>
+
+          {/* graph section */}
+          <ParentSize>
+            {({ width, height }) => {
+              const squareSize = Math.min(width, height);
+
+              // simulate shift key being pressed
+              let shiftDownEvent = new KeyboardEvent('keydown', {
+                key: 'Shift',
+                keyCode: 16,
+                code: 'ShiftLeft',
+                location: 1,
+                ctrlKey: false,
+                shiftKey: true,
+                altKey: false,
+                metaKey: false,
+                bubbles: true,
+              });
+              document.dispatchEvent(shiftDownEvent);
+
+              return (
+                <Stack justifyContent="space-between" overflow={"hidden"} padding={1} sx={{ border: '2px solid', borderColor: 'grey.400', borderRadius: '8px', height: '57vh'}}>
+                  <Stack direction="row" justifyContent="space-between" mt={1} sx={{ backgroundColor: '#dbdefc', borderRadius: '8px', zIndex: 10 }}>
```

Comment:
Design doesn't have this appear when no samples are selected. IMO It's not obvious that you can even select in the first place so I'm fine with keeping this, if only to suggest that a selection can be made